consensus/ethash/sealer.go

```diff
@@ -226,11 +231,15 @@ func (ethash *Ethash) remote(notify []string) {
 	// submitWork verifies the submitted pow solution, returning
 	// whether the solution was accepted or not (not can be both a bad pow as well as
 	// any other error, like no pending work or stale mining result).
-	submitWork := func(nonce types.BlockNonce, mixDigest common.Hash, hash common.Hash) bool {
+	submitWork := func(nonce types.BlockNonce, mixDigest common.Hash, sealhash common.Hash) bool {
 		// Make sure the work submitted is present
-		block := works[hash]
+		block := works[sealhash]
 		if block == nil {
-			log.Info("Work submitted but none pending", "hash", hash)
+			log.Info("Work submitted but none pending", "sealhash", sealhash, "current block number", currentBlock.NumberU64())
+			return false
+		}
+		if currentBlock == nil {
+			log.Error("have pending packages but no pending block, please file an issue", "sealhash", sealhash)
```

Comment:
"have pending packages but no pending block, please file an issue" -> "Pending work without block". I don't think we need to explicitly ask people to file an issue. The big red `ERROR` will be enough for them to know something's very off.